twine/settings.py

```diff
@@ -223,6 +229,13 @@ def register_argparse_arguments(parser):
             action="store_true",
             help="Show verbose output."
         )
+        parser.add_argument(
+            "-v", "--verify",
+            default=True,
+            required=False,
+            action="store_false",
```

Comment:
This usage seems counter-intuitive to me. If I understand correctly, passing `--verify` will disable verification. Probably better would be `--verify={yes/on/true,no,off/false}` and have a default (`yes`).